uiowa/uiowa01
# Feature: Entity view managed by Layout Builder

For all entity types except Deadline, the management of the display is now controlled via Layout Builder. Additionally, there has been some other clean-up around the entity_types_* modules as documented in the associated commits.

# How to test
1. `drush @theming.local si collegiate`
2. Install all entity_types_* modules except Deadline.
3. Visit "Manage Display" for each content type and verify that Layout Builder is enabled. 
4. Also, visit the "Manage Layout" page for each content type and confirm that the page title and breadcrumbs are in their own section above the rest of the fields.
